rust_src/src/indent.rs

```diff
@@ -134,4 +141,46 @@ pub extern "C" fn invalidate_current_column() {
     unsafe { last_known_column_point = 0 };
 }
 
+/// Indent from point with tabs and spaces until COLUMN is
+/// reached. Optional second argument MINIMUM says always do at least
+/// MINIMUM spaces even if that goes past COLUMN; by default, MINIMUM
+/// is zero.
+///
+/// The return value is the column where the insertion ends.
+#[lisp_fn(min = "1")]
```

Comment:
You missed the `intspec`. rust_src/rustize-remacs is a helper script you can call with `M-|` to automatically port the DEFUN comments and make a new lisp_fn for you. Give it a try.

Reply:
awesome :+1: